code/src/UI/Controls/CodeViewer.cs

```diff
@@ -11,6 +11,7 @@
 
 using Microsoft.Templates.UI.Services;
 using Microsoft.Templates.UI.ViewModels.NewItem;
+using Microsoft.Templates.UI.ViewModels.Common;
```

Comment:
I strongly believe we will need to make this a common shared item after we do some refactoring. It was tightly tied to a VM to right click add